doc/migration/version-upgrade-guide.md

```diff
@@ -810,6 +810,43 @@ This release contains breaking changes in some commands.
 
 See the [CHANGELOG](https://github.com/operator-framework/operator-sdk/blob/master/CHANGELOG.md#v0151) for details of the release.
 
+## Unreleased
+
+### Breaking changes
+
+#### Remove Ansible container sidecar
+
+The additional of the dependency `inotify-tools` on Ansible based-operator images is deprecated and it will be removed in the next versions. In this way, update the `deploy/operator.yaml` file as follows.
+
+Remove:
+
+```
+- name: ansible
+  command:
+    - /usr/local/bin/ao-logs
+    - /tmp/ansible-operator/runner
+    - stdout
+  # Replace this with the built image name
+  image: "REPLACE_IMAGE"
+  imagePullPolicy: "Always"
+  volumeMounts:
+    - mountPath: /tmp/ansible-operator/runner
+    name: runner
+    readOnly: true
+```
+
+Replace:
+
+```yaml 
+- name: operator
+```
+
+With:
+
+```yaml 
+- name: {{your operator name which is the value of metadata.name in this file}}
+```
+
```

Comment:
@estroz I followed here your suggestion. 
I will try to start to do that as well. 
Let's see if it can build a good example/case in the order we start to do it for all.